charts/rekor-tiles/templates/configmap.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if .Values.signer.tink }}
```

Comment:
JW, why a configmap for tink and not for the file signer?

Reply:
Oh I think I answered my question, it's a secret rather than a configmap, yea?

Reply:
Trying to mirror what TSA does https://github.com/sigstore/helm-charts/blob/main/charts/tsa/templates/tsa-configmap.yaml

Rekor v1 doesn't seem to have tink support in its helm chart so I didn't have a better example to go from.

No it's not a secret, if it should be I can change it (that's another ExternalSecret we'd need to manage).

Reply:
Right, it will be a private key and possibly also a password. I only added the secret at all to make CI happy since it depended on the secret existing, otherwise this should be created out of band in a more secure way.